pact/provider/assertj/src/main/java/org/arquillian/algeron/pact/provider/assertj/PactProviderAssertions.java

```diff
@@ -0,0 +1,29 @@
+package org.arquillian.algeron.pact.provider.assertj;
+
+import org.arquillian.algeron.pact.provider.core.httptarget.Target;
+import org.assertj.core.api.Assertions;
+
+import java.net.URL;
+
+
+public class PactProviderAssertions extends Assertions {
+
+    Target target;
+
+    PactProviderAssertions(Target target) {
+        this.target = target;
+    }
+
+    public static PactProviderAssertions assertThat(Target target) {
+        return new PactProviderAssertions(target);
+    }
+
+    public void verifiesContract() {
+        this.target.testInteraction();
+    }
+
+    public ProviderUrlAssert withUrl(URL url) {
+        return new ProviderUrlAssert(this, url);
```

Comment:
![image](https://cloud.githubusercontent.com/assets/719616/20724323/0cf54b00-b66e-11e6-998f-fe058b31f5e3.png)

From the user perspective that would be way more convenient.